openbb_terminal/API_README.md

```diff
@@ -1,84 +1,280 @@
-# OpenBB API
+# OpenBB SDK
 
-## Usage
+OpenBB SDK gives you direct and programmatic access to all capabilities from OpenBB Terminal. You will have the necessary building blocks to create your own financial tools and applications, whether that be a visualization dashboard or a custom report on Jupyter Notebook. With OpenBB SDK, you can access to normalized financial data from dozens of data providers, without having to develop your own integrations from scratch. On top of financial data feeds, OpenBB SDK also provides you with a toolbox to perform financial analysis on a variety of asset classes, including stocks, crypto, ETFs, funds; the economy as well as your portfolios.
 
-The OpenBB API can simply be imported with:
+OpenBB SDK is created and maintained by OpenBB team together with the contributions from hundreds of community members. This gives us an unrivaled speed of development and the ability to maintain stable integrations with numerous third-party data providers. Developing and maintaining an full-blown investment research infrastructure from the ground up takes a lot of time and effort. However, it does not have to be. Take advantage of OpenBB SDK with its out-of-the-box data connectors and financial analysis toolkit; so that you can focus on designing and building your financial reports and applications.
 
-```python
+## Installation
+
+To be completed once pip install is available on PyPI
```

Comment:
@simmonsj330 can you help document on how to do pip install when you list on the actual PyPi? Thank you! 🙌

Reply:
@minhhoang1023 Should we add in the installation method in now but have a comment or something saying "Not yet available" ?

Reply:
@simmonsj330 okay let's add it later then